Sources/Paywalls/PaywallData.swift

```diff
@@ -15,6 +15,25 @@
 
 import Foundation
 
+/// Represents countries where currencies typically have zero decimal places
+public struct ZeroDecimalPlaceCountries: Codable, Sendable, Hashable, Equatable {
+
+    /// Storefront country codes that should typically display zero decimal places
+    public var apple: Set<String>
```

Comment:
Can this just be 👇 

```suggestion
    public var apple: [String]
```

This would allow us to not have a custom decoder 🤷‍♂️

Reply:
`Set<String>` actually works without the custom decoder - the reason for the custom decoder is so that if the key `zero_decimal_place_countries` is there, but it doesn't contain `apple`, it will still decode `apple` to an empty Set rather than fail to decode and throw. Is defensively coding for a missing `apple` worthwhile? Or is there a better way? Thoughts?

Reply:
Went to `[String]` because order is important for test validation (json snapshot comparing). Realistically there won't be any performance difference.